openoa/plant.py

```diff
@@ -1259,7 +1259,7 @@ def __attrs_post_init__(self):
 
         if self.scada is not None:
             self.calculate_turbine_energy()
-
+        
```

Comment:
Looks like the pre-commit workflow didn't run based on this extra whitespace, could you just make sure that it runs and passes?

Reply:
I ran the precommit workflow.